packages/backend/package.json

```diff
@@ -9,6 +9,7 @@
     "build": "nest build --watch --webpack",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
     "start": "nest build & node dist/main",
+    "start1": "node dist/main",
```

Comment:
czemu to pozmienialas nie bylo zwiazane z twoim zadaniem?

Reply:
ok